modules/upgrade/pages/proxy-migration.adoc

```diff
@@ -41,7 +41,7 @@ mount -o loop DVD1.iso /mnt/sle15
 . In the [guimenu]``Create Autoinstallable Distribution`` section, use these parameters:
 * In the [guimenu]``Distribution Label`` section, type a unique name for the distribution.
 Use only letters, numbers, hyphens (``-``), periods  (``.``), and underscores (``_``), and ensure the name is longer than four characters.
-* In the [guimenu]``Tree Path`` field, type an absolute path to the installation source.
+* In the [guimenu]``Tree Path`` field, type an absolute path to the installation source. In this example, this would be ``/mnt/sle15sp1``
```

Comment:
```suggestion
* In the [guimenu]``Tree Path`` field, type an absolute path to the installation source.
In this example, this would be [path]``/mnt/sle15sp1``.
```

Reply:
Ahh, yes!